templates/Builder/Om/baseQueryClassHeader.php

```diff
@@ -77,12 +77,12 @@
  * @method     <?= $modelClass ?> requireOneBy<?= $column->getPhpName() ?>(<?= $column->getPhpType() ?> $<?= $column->getName() ?>) Return the first <?= $modelClass ?> filtered by the <?= $column->getName() ?> column and throws <?= $entityNotFoundExceptionClass ?> when not found
 <?php endforeach;?>
  *
- * @method     <?= $modelClass ?>[]|Collection find(?ConnectionInterface $con = null) Return <?= $modelClass ?> objects based on current ModelCriteria
- * @psalm-method Collection&\Traversable<<?= $modelClass ?>> find(?ConnectionInterface $con = null) Return <?= $modelClass ?> objects based on current ModelCriteria
+ * @method     <?= $modelClass ?>[]|\Propel\Runtime\Collection\Collection find(?ConnectionInterface $con = null) Return <?= $modelClass ?> objects based on current ModelCriteria
```

Comment:
Standard Phpdoc is fine with this notation, as there is already a declaraction above in use.
Please update only psalm.